site/content/3.12/release-notes/version-3.12/api-changes-in-3-12.md

```diff
@@ -48,6 +48,13 @@ unknown attributes and values are detected in the request body.
 Note that all invalid elements and combinations will be rejected in future
 versions.
 
+#### Dropping a collection from a graph
+
+Dropping a collection from a graph using the
+`DELETE /_api/collection/{collection-name}` endpoint is now strictly enforced
+so that the graph definition remains intact. Previously, it was allowed to drop
```

Comment:
Reword slightly to avoid potential confusion (Dropping a collection ... is strictly enforced)
```suggestion
`DELETE /_api/collection/{collection-name}` endpoint now strictly enforces
that graph definitions remain intact. Previously, it was allowed to drop
```